__tests__/components/editor/PropertyComponent.test.js

```diff
@@ -62,31 +60,28 @@ describe('<PropertyComponent />', () => {
       it('renders the lookup component', () => {
         expect(wrapper.find('Connect(InputLookupQA)').length).toEqual(1)
       })
-
     })
   })
 
   describe('when there are no configuration values from the property template', () => {
-
     describe('for unconfigured templates of type:literal', () => {
       const template = {
-        "propertyURI": "http://id.loc.gov/ontologies/bibframe/heldBy",
+        "propertyURI": "http:id.loc.gov/ontologies/bibframe/heldBy",
```

Comment:
Is this intentional?

Reply:
no! I didn't even see that. Will change. Thanks!

Reply:
done. pushed. resolving.